src/pages/Detail/ProductInfo/ProductInfo.jsx

```diff
@@ -1,8 +1,7 @@
-import { useEffect, useState } from 'react';
+import { useState } from 'react';
 import * as S from './ProductInfo.Style';
 
-export default function ProductInfo() {
-  const [productInfo, setProductInfo] = useState([{}]);
+export default function ProductInfo({ productDetail }) {
```

Comment:
```suggestion
export default function ProductInfo({ productDetail:{
productName,ratingAmount,productPrice,discountPercentage 등등등
} }) {
```
구조분해할당 바로 가능합니다. 참고 해 주세요~

Reply:
넵 알겠습니다!